recipe/meta.yaml

```diff
@@ -1,38 +1,22 @@
 {% set name = "zeromq" %}
-{% set version = "4.2.1" %}
-{% set sha256 = "27d1e82a099228ee85a7ddb2260f40830212402c605a4a10b5e5498a7e0e9d03" %}
+{% set version = "4.2.3" %}
+#{% set sha256 = "8f1e2b2aade4dbfde98d82366d61baef2f62e812530160d2e6d0a5bb24e40bc0" %}
+{% set sha256 = "b428c6cdf1df4b5cdcb3a6727c6ece85c7fb05d7907c532566a115b4dda113a8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/zeromq/libzmq/releases/download/v{{ version }}/zeromq-{{ version }}.tar.gz
-
+  #url: https://github.com/zeromq/libzmq/releases/download/v{{ version }}/zeromq-{{ version }}.tar.gz
+  url: https://github.com/zeromq/libzmq/archive/v{{ version }}.tar.gz
```

Comment:
No need to keep the commented version. Merging so that we don't need to wait for the whole CI another time.

Reply:
The commented version should have been kept. Building from git-archive omits sdist-related files required for standard builds.

Reply:
Note: The reason for using git archive is that the generated sdist release lacks content required for cmake build.

Responding on the general issue in the open GH issue.